src/services/textChanges.ts

```diff
@@ -28,17 +28,32 @@ namespace ts.textChanges {
     }
 
     export interface ConfigurableStart {
-        /** True to use getStart() (NB, not getFullStart()) without adjustment. */
-        useNonAdjustedStartPosition?: boolean;
+        startPosition?: LeadingTriviaOption;
     }
     export interface ConfigurableEnd {
         /** True to use getEnd() without adjustment. */
-        useNonAdjustedEndPosition?: boolean;
+        endPosition?: TrailingTriviaOption;
     }
 
-    export enum Position {
-        FullStart,
-        Start
+    export enum LeadingTriviaOption {
+        /** Exclude all leading trivia (use getStart()) */
+        Exclude,
+        /** Include leading trivia (default behavior) */
+        Include,

```

Comment:
Is this used?

Reply:
I've removed the default options from `LeadingTriviaOption` and `TrailingTriviaOption`.